### ISSUE TITLE
Excluding node_modules in watcher

### ISSUE THREAD

**Issue by guybedford** [amasad/sane#72]
If running the Node native watcher on a folder that contains a large node_modules directory, we want the ability to ignore this folder.

Is there any way in this project (or with the Node native watcher for that matter), to handle use cases like this?

See https://github.com/jspm/jspm-cli/issues/1999 for more info.

**Comment by amasad** [comment]
Have you tried the globs option?

**Comment by guybedford** [comment]
See the discussion previously at https://github.com/jspm/jspm-cli/issues/1775#issuecomment-217046015.

The issue is that because we know the specific module tree, we do know exactly which individual files to watch.

I'm wondering if it's worth just falling back to a custom `fs.watch` call here on the specific directories we know we need to listen to, as we don't need recursion actually.

**Comment by amasad** [comment]
Yeah that might make sense. We can also have a predicate function that gets called on directories and if returns false it will skip watching that dir or any of it's descendants. I don't have the time to do any work on this. So let me know what works for you and patches welcome
